### PR TITLE
Add Finnish translation

### PR DESCRIPTION
**Pull request by rutkuli**
Added finnish translation for installer.

### CODE REVIEW

**Comment by amd64fox** [comment]
Hello, thanks for the translation
could you type this into the powershell terminal:
```ps1
$PSUICulture
```
and send output here

**Comment by rutkuli** [comment]
fi-FI

File: scripts/installer-lang/fi.ps1 [hunk 1]
```diff
@@ -0,0 +1,61 @@
+[PSCustomObject]@{
+    Welcome         = "
+╔══════════════════════════════╗
+║ Welcome to SpotX for Windows ║
```
Comment (amd64fox) [inline scripts/installer-lang/fi.ps1]: Will the logo remain in English or will you also translate it?

Comment (rutkuli) [reply scripts/installer-lang/fi.ps1]: I changed it.

Comment (amd64fox) [reply scripts/installer-lang/fi.ps1]: place text in the center

Comment (rutkuli) [reply scripts/installer-lang/fi.ps1]: Sorry, I didn't see that. Should be fixed now.

**Pull request merged**